### PR TITLE
Issue 1252 pass phrase in ram create msg

### LINKED ISSUE #1252: Pass phrase in memory. Creat an encrypted message
as a part of #372

### PR DESCRIPTION
**Pull request by DenBond7**
This PR added handling passphrases in RAM when a user is creating a message

close #1252
----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities

### CODE REVIEW

**Comment by DenBond7** [comment]
@tomholub this issue blocks me to continue work on #1253 and #1254. If you have some free time, please reviews it. Otherwise, I will work on another task that is not related to `passphrase in RAM`.

**Comment by tomholub** [comment]
Very sorry - I'm completely swamped this week due to https://flowcrypt.com/blog/article/2021-06-14-google-workspace-encryption/ so I won't be of much help for reviewing (but will still answer product direction questions). Either @seisvelas or @IvanPizhenko should be able to get to it soon.

**Comment by DenBond7** [comment]
> Very sorry - I'm completely swamped this week due to https://flowcrypt.com/blog/article/2021-06-14-google-workspace-encryption/ so I won't be of much help for reviewing (but will still answer product direction questions). Either @seisvelas or @IvanPizhenko should be able to get to it soon.

No prob. I'm already working on #1168

File: FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/dialog/FixNeedPassphraseIssueDialogFragment.kt [hunk 1]
```diff
@@ -237,7 +238,16 @@ class FixNeedPassphraseIssueDialogFragment : BaseDialogFragment() {
     })
   }
 
+  private fun sendResult(result: Int) {
+    if (targetFragment == null) {
+      return
+    }
+    targetFragment?.onActivityResult(targetRequestCode, result, null)
```
Comment (IvanPizhenko) [inline FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/dialog/FixNeedPassphraseIssueDialogFragment.kt:245]: could be a bit simpler: `if (targetFragment != null) { ... }`

**Review by IvanPizhenko — APPROVED** [review]

**Pull request merged**